code/game/gamemodes/blob/theblob.dm

```diff
@@ -110,7 +114,9 @@
 
 /obj/effect/blob/proc/expand(turf/T = null, prob = 1, a_color)
 	if(prob && !prob(health))	return
-	if(istype(T, /turf/space) && prob(75)) 	return
+	var/blobtype = /obj/effect/blob/normal
+	if(istype(T, /turf/space))
+		blobtype = /obj/effect/blob/normal/fragile

```

Comment:
this shouldn't be a type, it should just set the vars on the blob tile created, since you already put the logic for fragile/healed blobs there.